src/Env/Config/Renderer.php

```diff
@@ -40,45 +37,12 @@ public static function render(Config $config)
         }
 
         $vars = $config->getVars();
-        $content = file_get_contents($template);
-
-        if ('yml' === $template->getExtension()) {
-            return self::renderYaml($content, $vars);
-        }
-
-        $rendered = $content;
+        $rendered = file_get_contents($template);
 
         foreach ($vars as $var) {
```

Comment:
We could use `array_reduce` in order to inline the `return`. But this should be benchmarked. Nobody ain't time for that. :trollface: